www/install/steps/process/process_step6.php

```diff
@@ -94,13 +96,14 @@
     }
 }
 
-try {
-    checkMariaDBPrerequisite($link);
-} catch (\Exception $e) {
-    $err['connection'] = $e->getMessage();
+if ($link !== null) {
+    try {
+        checkMariaDBPrerequisite($link);
```

Comment:
Maybe you could do this check inside the previous try/cacth block ?

Reply:
@TamazC inside the `catch` ?
Because this issue happens when there is a `PDOException`